### DISCUSSION TITLE
detekt 2.0 - Call to Action

### DISCUSSION THREAD

**Discussion by cortinico** [detekt/detekt#5998] [Announcements]
Hi all,
I'm opening this discussion to collect interest for folks interested in **detekt 2.0**.

We've been planning the list of features we decided to include inside 2.0 and we'll be sharing more on the detekt 2.0 roadmap after KotlinConf. In the coming future, we'll come up with a list of **actionable tasks** that folks can pick up and execute on.

This is a great opportunity to join the conversation and **contribute** to detekt.

If you're interested, comment below or subscribe to notification to this discussion, so you'll be notified as we do have more information to share.

On behalf of the detekt maintainers,
— Nicola

**Comment by PoisonedYouth** [comment]
Hey Nicola,
if you need support for making detekt ready for the 2.0 major version, I can support.

**Comment by cortinico** [comment]
UPDATE: Hey all,
We've posted an update with the actual plans for 2.0 here:
- https://github.com/detekt/detekt/discussions/6035
